sopel/cli/plugins.py

```diff
@@ -319,8 +319,12 @@ def handle_configure(options):
         return 0  # nothing to configure is not exactly an error case
 
     print('Configure %s' % plugin.get_label())
-    plugin.configure(settings)
-    settings.save()
+    try:
+        plugin.configure(settings)
+        settings.save()
+    except KeyboardInterrupt:
+        tools.stderr('\nOperation cancelled, no file has been modified.')
```

Comment:
```suggestion
        tools.stderr('\nOperation cancelled; the config file has not been modified.')
```

Comma splice bad, redux. Also weird phrasing this time.